CMakeLists.txt

```diff
@@ -18,6 +18,39 @@ option(OTURB_ENABLE_ALL_WARNINGS "Show most warnings for most compilers" ON)
 # option(OTURB_ENABLE_CPPCHECK "Enable cppcheck static analysis target" OFF)
 # option(OTURB_ENABLE_FCOMPARE "Enable building fcompare when not testing" OFF)
 
+# Set RELEASE as default build type if none provided
+set(SUPPORTED_BUILD_TYPES None Debug Release RelWithDebInfo MinSizeRel)
+string(REPLACE ";" " " SUPPORTED_BUILD_TYPES_STRING "${SUPPORTED_BUILD_TYPES}")
```

Comment:
> What would the "None" build type be?

Thanks for catching this, it remained from a previous design I was considering for this.

> Also, just heads up that the options provided are the defaults from CMake so this may be redundant.

Based on the reply to your previous comment on line 22, does the list make sense? I might be missing something here.